src/components/farm/FarmLatestBlockChallenges.tsx

```diff
@@ -67,6 +67,7 @@ export default function FarmLatestBlockChallenges() {
             color="primary"
             href="https://www.chiaexplorer.com/"
             target="_blank"
+            to="/"
```

Comment:
we need to fix component prop types and to add unused property

Reply:
Could you lend me a hand with that?

Reply:
> Could you lend me a hand with that?

i think i got it

Reply:
Let me know what you think